engine/lib/phx/src/physics/physics.rs

```diff
@@ -266,12 +266,14 @@ impl Physics {
             iterator.index += 1;
 
             // Evaluate contact.
-            let c1_parent = RigidBody::linked_with_collider_mut(
-                world.colliders.get(contact_pair.collider1).unwrap(),
-            );
-            let c2_parent = RigidBody::linked_with_collider_mut(
-                world.colliders.get(contact_pair.collider2).unwrap(),
-            );
+            let c1_parent = world
+                .colliders
+                .get(contact_pair.collider1)
+                .and_then(RigidBody::linked_with_collider_mut);
```

Comment:
`and_then` will call `RigidBody::linked_with_collider_mut` on the result of `.get` if `get` returns a `Some(...)`. If it returns a `None`, that will just get ignored and `c1_parent` will become `None` as well.

This is basically equivalent to:
```
                    if let Some(collider) = world.colliders.get(other_collider) {
                        RigidBody::linked_with_collider_mut(collider)
                    } else {
                        None
                    }
                    ```